### DISCUSSION TITLE
Would it be possible to skip the address validation during checkout?

### DISCUSSION THREAD

**Discussion by wilianto** [saleor/saleor#13370] [Q&A]
Hello Saleor team!

I want my customer to be able to choose "Other" as country area / province in billing address.
Would it be possible to skip the google-i18n address validation but keeping the field to be required?

Cheers!

**Comment by agsmartwolf** [comment]
So is it possible?

**Comment by slamer59** [comment]
As far as I know I am able to create a checkout with only an email adress for the customer. But I wrote my own checkout process.

**Comment by timuric** [comment]
Please take a look at https://docs.saleor.io/docs/3.x/developer/checkout/address
